### DISCUSSION TITLE
Restore Backup

### DISCUSSION THREAD

**Discussion by ddyer8** [tagyoureit/nodejs-poolController#505] [REM]
I'm still attempting to add functionality to my Nixie/REM setup.  To use influxDB v2, I'm migrating to the 64bit version of raspian.  I have the software up and running on a new sd card, but it seems I was unable to restore the configuration for REM.  Despite the warning not to restore from a backup unless instructed to do so, I attempted backup from my old card and restore to my new card.  It seems the nodejs-poolController restored, but I don't have any REM configuration.  The big thing I'm trying to avoid is recalibrating the atlas pH and ORP probes. Is it as simple as coping a config file from my old 32bit sd card to my new system?

**Comment by rstrouse** [comment]
The calibration is stored on the EZO-PH circuit.  Magically it will read it back from the card.

That being said you can simply copy the file.

**Reply by ddyer8** [reply]
Sweet.  Makes sense.  I've got it up and running.  I never was able to find the right config files.  Some settings came over, but I had to setup REM from scratch.  Wasn't that big of a deal though.
